rclrs/build.rs

```diff
@@ -3,8 +3,23 @@ use std::fs::read_dir;
 use std::path::{Path, PathBuf};
 
 const AMENT_PREFIX_PATH: &str = "AMENT_PREFIX_PATH";
+const ROS_DISTRO: &str = "ROS_DISTRO";
+
+fn get_env_var_or_abort(env_var: &'static str) -> String {
+    if let Ok(value) = env::var(env_var) {
+        value
+    } else {
+        panic!(
+            "{} environment variable not set - please source ROS 2 installation first.",
+            env_var
+        );
+    }
+}
 
 fn main() {
+    let distro = get_env_var_or_abort(ROS_DISTRO);
+    println!("cargo:rustc-cfg=distro=\"{distro}\"");
```

Comment:
Could you rename this to `ros-distro` or `ros_distro`? I know it's nitpicky :sweat_smile:

Reply:
No problem, done.

Reply:
Thanks.